### PR TITLE
[eclipse-marketplace] New last updated Eclipse badge

### PR DESCRIPTION
**Pull request by PyvesB**
This pull request implements a fifth Eclipse Marketplace badge, which displays the "date" when the plugin was last updated on the marketplace. For instance:

last updated

Cheers,

Pyves

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -5392,7 +5392,7 @@ camp.route(/^\/vscode-marketplace\/(d|v|r)\/(.*)\.(svg|png|gif|jpg|json)$/,
 );
 
 // Eclipse Marketplace integration.
-camp.route(/^\/eclipse-marketplace\/(dt|dm|v|favorites)\/(.*)\.(svg|png|gif|jpg|json)$/,
+camp.route(/^\/eclipse-marketplace\/(dt|dm|v|favorites|updated)\/(.*)\.(svg|png|gif|jpg|json)$/,
```
Comment (paulmelnikow) [inline server.js]: Since the github badge calls this `last-update`, how about using the same thing here for consistency?

File: try.html [hunk 2]
```diff
@@ -2089,12 +2089,19 @@ <h3 id="miscellaneous">Miscellaneous</h3>
           <td><code>https://img.shields.io/vscode-marketplace/r/ritwickdey.LiveServer.svg</code></td>
         </tr>
         <tr>
-          <th data-keywords="eclipse marketplace">Eclipse Marketplace:</th>
+          <th data-keywords="eclipse marketplace favorites">Eclipse Marketplace favorites:</th>
```
Comment (paulmelnikow) [inline try.html]: After #1163 `try.html` is generated, so please make these changes in [lib/all-badge-examples.js](https://github.com/badges/shields/blob/master/lib/all-badge-examples.js) instead, and then run `npm run build`.

`try.html` will no longer be tracked after #1194.

Comment (PyvesB) [reply try.html]: Done, will keep that in mind in the future!

**Comment by paulmelnikow** [comment]
Thanks!

**Pull request merged**